libs/blocks/table/table.js

```diff
@@ -598,6 +598,16 @@ function applyStylesBasedOnScreenSize(table, originTable) {
   setRowStyle();
 }
 
+function handleStickyHeader(el) {
+  if (!el.classList.value.includes('sticky')) return;
+
+  setTimeout(() => {
+    const headingRow = el.querySelector('.row-heading');
+    if (headingRow.offsetHeight / window.innerHeight < 0.45) el.classList.remove('cancel-sticky');
+    if (headingRow.offsetHeight / window.innerHeight >= 0.45) el.classList.add('cancel-sticky');
```

Comment:
Micro-nit, how about
```suggestion
    el.classList.toggle('cancel-sticky', !(headingRow.offsetHeight / window.innerHeight < 0.45));
```